en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -8,88 +8,94 @@ Performing root cause analysis is crucial in identifying and rectifying the unde
 ## Get started with the sample service
 
 1. Go to **Services** from the left navigation.
-2. If this is your first service, then enter a name for the service and click **Create**. Skip step 3 and go to step 4.
-       ![Create a new service](../assets/img/observability/first-service-creation.png)
-3. Else, click **+ Create** to create a new service.
-       ![Create a new service](../assets/img/observability/service-creation.png)
-    1. Enter a name for the service and click **Create**.
-4. From the left-hand menu, click **Observe**.
-5. On the right-hand bottom, click the **Sample Service** icon.
+2. Create a new service.
+    1. **If this is your first service**, 
+           1. Enter a name for the service and click **Create**. 
+           ![Create a new service](../assets/img/observability/first-service-creation.png)                        
+    2. **Else** (if you already have services created), 
+           1. Click **+ Create** to start creating a new service.
+           2. Enter a name for the service and click **Create**.
+           ![Create a new service](../assets/img/observability/service-creation.png)
```

Comment:
It seems this is not rendering properly. 
<img width="997" alt="image" src="https://user-images.githubusercontent.com/16300038/118082036-1e729400-b3da-11eb-9316-077c4ec0b3e8.png">

Reply:
Fixed.